tests/client/admin/containers/TestCurrentSearchPage.js

```diff
@@ -1,161 +1,168 @@
-  });
-
-  it('treats 0 as 1', () => {
-    assert.strictEqual(parsePage(0), 1);
-  });
-
-  it('treats empty strings as 1', () => {
-    assert.strictEqual(parsePage(''), 1);
-  });
-});
-
-describe('CurrentSearchPage.loadSearchResultsIfNeeded()', () => {
-  it('does not dispatch on undefined query', () => {
-    const dispatchSpy = sinon.spy();
-    const state = { loading: false };
-    const store = { dispatch: dispatchSpy, getState: () => ({ search: state }) };
-    const location = { query: { page: undefined, q: undefined } };
-    loadSearchResultsIfNeeded({ store, location });
-    assert.notOk(dispatchSpy.called);
-  });
-
-  it('returns right away when loaded', () => {
-    const page = 10;
-    const query = 'no ads';
-    const state = { loading: false, page, query };
-    const store = { dispatch: sinon.spy(), getState: () => ({ search: state }) };
-    const location = { query: { page, q: query } };
-    assert.strictEqual(loadSearchResultsIfNeeded({ store, location }), true);
-  });
-
-  it('loads the search results when needed', () => {
-    const page = 10;
-    const query = 'no ads';
-    const state = {
-      api: { token: 'a.jwt.token' },
-      search: { loading: false, page, query: 'old query' },
-    };
-    const dispatch = sinon.spy();
-    const store = { dispatch, getState: () => state };
-    const location = { query: { page, q: query } };
-    const mockApi = sinon.mock(api);
-    const entities = sinon.stub();
-    const result = sinon.stub();
-    mockApi
-      .expects('search')
-      .once()
-      .withArgs({ page, query, api: state.api })
-      .returns(Promise.resolve({ entities, result }));
-    return loadSearchResultsIfNeeded({ store, location }).then(() => {
-      mockApi.verify();
-      assert(
-        dispatch.firstCall.calledWith(actions.searchStart(query, page)),
-        'searchStart not called');
-      assert(
-        dispatch.secondCall.calledWith(actions.searchLoad({ query, entities, result })),
-        'searchLoad not called');
-    });
-  });
-
-  it('triggers searchFail when it fails', () => {
-    const page = 11;
-    const query = 'no ads';
-    const state = {
-      api: {},
-      search: { loading: false, page, query: 'old query' },
-    };
-    const dispatch = sinon.spy();
-    const store = { dispatch, getState: () => state };
-    const location = { query: { page, q: query } };
-    const mockApi = sinon.mock(api);
-    mockApi
-      .expects('search')
-      .once()
-      .withArgs({ page, query, api: state.api })
-      .returns(Promise.reject());
-    return loadSearchResultsIfNeeded({ store, location }).then(() => {
-      mockApi.verify();
-      assert(
-        dispatch.firstCall.calledWith(actions.searchStart(query, page)),
-        'searchStart not called');
-      assert(
-        dispatch.secondCall.calledWith(actions.searchFail({ page, query })),
-        'searchFail not called');
-    });
-  });
-});
+// import * as searchActions from 'core/actions/search';
```

Comment:
Actually just checked, this was shared between admin/amo so it's in core now. I should delete this file, sorry.